src/components/SearchBar.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { Icon } from "@iconify/react";
+
+import StyledTransparentButton from "../components/shared/StyledTransparentButton";
+
+export default function SearchBar({ onSubmitKeyword }) {
+  const [tag, setTag] = useState("");
+  const [errorMessage, setErrorMessage] = useState(false);
+
+  function handleChangeTag(tag) {
+    /* eslint-disable */
+    const reg = /[`~!@#$%^&*()_|+\-=?;:'",.<>\{\}\[\]\\\/ ]/gim;
+    /* eslint-enable */
+    if (tag.search(reg) > -1) {
+      setErrorMessage("특수문자 또는 공백은 입력할 수 없습니다.");
+    }
```

Comment:
```suggestion    
    const reg = /[`~!@#$%^&*()_|+\-=?;:'",.<>\{\}\[\]\\\/ ]/gim;

    if (tag.search(reg) > -1) {
      setErrorMessage("특수문자 또는 공백은 입력할 수 없습니다.");
    }
```
정규표현식 관련하여 eslint 오류인 경우 추후에 eslint 설정을 변경하는게 좋겠네요!

Reply:
상민님 말씀처럼 다음번에 설정해주면 좋을 것 같네요!
[eslint-plugin-regex](https://www.npmjs.com/package/eslint-plugin-regex) 정규식 관련 플러그인 입니다!